### PR TITLE
Add rolebinding-reader

### PR DESCRIPTION
**Pull request by sdodson**
Per https://github.com/openshift/origin-aggregated-logging/commit/d40fd9b83a6f6c0fb962db8869e6a35ef4f72751 a new role needs to be applied to aggregated-logging-elasticsearch

Fixes Bug 1390913
Fixes BZ1390913

### CODE REVIEW

**Comment by sdodson** [comment]
@richm Mind giving this a quick review?

File: roles/openshift_hosted_logging/tasks/deploy_logging.yaml [hunk 1]
```diff
@@ -71,6 +71,13 @@
     register: fluentd2_output
     failed_when: "fluentd2_output.rc == 1 and 'exists' not in fluentd2_output.stderr"
 
+  - name: "Add rolebinding-reader to aggregated-logging-elastic-search"
```
Comment (richm) [inline roles/openshift_hosted_logging/tasks/deploy_logging.yaml]: `elasticsearch`

**Comment by richm** [comment]
one minor nit - otherwise LGTM

**Comment by sdodson** [comment]
aos-ci-test

**Comment by richm** [comment]
Are you planning to cherry-pick this to 3.4 or 1.4?  This feature https://github.com/openshift/origin-aggregated-logging/commit/d40fd9b83a6f6c0fb962db8869e6a35ef4f72751 depends on it.  OTOH, is the `rolebinding-reader` in the origin 1.4 release?  It hasn't been in the 3.4 pre-release code I've been testing with.  @ewolinetz

**Comment by sdodson** [comment]
Yeah, this will be pulled into release-1.4.

**Pull request merged**